opt/login-form/login.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Rice Lake Login</title>
```

Comment:
```suggestion
    <title>Login</title>
```

Only tweak would be to generalize this!